docs/docusaurus/docs/core/expectation_suites/manage_expectation_suites.md

```diff
@@ -0,0 +1,294 @@
+---
+sidebar_label: 'Manage Expectation Suites'
+title: 'Manage Expectation Suites'
+description: Create and manage sets of Expectations in Python with GX Core.
+---
+
+An Expectation Suite contains a group of Expectations that describe the same set of data.  All of the Expectations that you apply to your data will grouped into Expectation Suites.
+
+## Prerequisites
+
+This guide assumes you have:
+
+- Installed Python.
```

Comment:
```suggestion
- Python (https://www.python.org/downloads/)
```
Is there a minimum version?

Reply:
There is -- I intend to replace prerequisites like this that are used repeatedly with an imported line for convenience with keeping them standardized (plus so the version only needs updated in one place).  I'm planning to do that as part of the "revise OSS guides that don't have API changes" work.